servers/quarkus-config/src/main/java/org/projectnessie/quarkus/config/CatalogS3BucketConfig.java

```diff
@@ -17,17 +17,12 @@
 
 import io.smallrye.config.WithName;
 import java.util.Optional;
+import org.projectnessie.catalog.files.s3.S3AuthType;
 import org.projectnessie.catalog.files.s3.S3BucketOptions;
-import org.projectnessie.catalog.files.s3.S3ClientAuthenticationMode;
-import org.projectnessie.catalog.files.s3.S3ServerAuthenticationMode;
 
 public interface CatalogS3BucketConfig extends S3BucketOptions {
 
   @WithName("server-auth-mode")
```

Comment:
We could imho even remove the method override completely wdyt?

```suggestion
  @WithName("auth-type")
```